docs/book/migrating.md

```diff
@@ -0,0 +1,36 @@
+# Migration Guide
+
+Version 3 is the first major release of `laminas-view` and includes a number of backwards incompatible changes.
+
+## Helper Removals
```

Comment:
It would be good if we could create a constant outline for this page.

For example:

- New Features
- New Dependencies
- Signature Changes / Signature and Behavior Changes
- Removed Features
- Removed Classes and Traits
- …

The removed helpers can be added under "Removed Features" in a separate section and heading.